resources/style.css

```diff
@@ -48,6 +48,11 @@ button:hover {
     box-shadow: inset 0 -3px #ffffff;
 }
 
+/* combined with the configuration option hover=true, you can set a style for any module like this */
+widget.hover #pulseaudio {
+    background-color: #a37800;
```

Comment:
I specifically chose this color as it is slightly darker than the already set default color for the pulseaudio module.